scene/3d/navigation_obstacle_3d.cpp

```diff
@@ -346,6 +351,15 @@ bool NavigationObstacle3D::get_carve_navigation_mesh() const {
 	return carve_navigation_mesh;
 }
 
+PackedStringArray NavigationObstacle3D::get_configuration_warnings() const {
+	PackedStringArray warnings = Node::get_configuration_warnings();
```

Comment:
```suggestion
	PackedStringArray warnings = Node3D::get_configuration_warnings();
```
Should be called on the immediate parent class

Reply:
Fixed, thanks!